src/perl/lib/wtsi_clarity/epp/reports/sample_qc_report.pm

```diff
@@ -227,7 +228,7 @@ sub _get_dna_amount_library_prep {
 
   # If the dna amount is not up to requirement, S.M. subtract 2ul from the cherrypick volume
   # (to bring the concentration up)...
-  if ($dna_amount < $requirement) {
+  if ($dna_amount < ($requirement - $BUFFER)) {
```

Comment:
Could you add a little comment about why are we subtract the BUFFER in the condition, please?